Source/SwiftLintFramework/Rules/PrivateUnitTestRule.swift

```diff
@@ -103,7 +114,19 @@ public struct PrivateUnitTestRule: ASTRule, ConfigurationProviderRule {
         -> [StyleViolation] {
 
             guard kind == .Class else { return [] }
-            let classViolations = validateClass(file, kind: kind, dictionary: dictionary)
+            guard isTestClass(dictionary) == true else { return [] }
```

Comment:
actually folding the `kind == .Class` check into `isTestClass(_:)` would be nice.